platform.spec.js

```diff
@@ -1,11 +1,12 @@
 describe('suite of tests for the platform',function () {
-  it('Router base js should not be changed', function() {
-    console.log('Unit Test: Router base js should not be changed.');
-    var checksum = objectHash(routerSetupConfig, { algorithm: 'md5', encoding: 'base64' });
-    expect(checksum).toBe('2f10vxKAVe5PHd17c8tJdA==');
+  it('Router base js\' status object should not be changed', function() {
+    console.log('Router base js\' status object should not be changed');
+    var checksum = md5(JSON.stringify(routerSetupConfig.status));
+    expect(checksum).toBe('66cf155c826f2251b2107067c39376a9');
   });
-  it('should run this example test', function() {
-    console.log('Unit Test: This sample test should run.');
-    expect(1).toBe(1);
+  it('Router base js\' routeTunnel() function should not be changed', function() {
+    console.log('Router base js\' routeTunnel() function should not be changed');
+    var checksum = md5(routerSetupConfig.routeTunnel.toString());
```

Comment:
toString() is being run on the routeTunnel() function to convert the function to a string.